.eslintrc

```diff
@@ -2,10 +2,6 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "plugins": ["flowtype"],
```

Comment:
On further checking, I think this relates to the latest version of eslint breaking the version of airbnb's style config we use. Is updating to the latest airbnb style config and matching eslint versions, etc. to that an option? Otherwise, we probably shouldn't extend the airbnb config at all.

Reply:
I was able to do that by moving eslint down to the latest point release of the previous major version. Long term we probably shouldn't extend airbnb but for now this will work.